internal/volume/volumes_information.go

```diff
@@ -92,13 +93,24 @@ type BackupVolumeInfo struct {
 	// Snapshot completes timestamp.
 	CompletionTimestamp *metav1.Time `json:"completionTimestamp,omitempty"`
 
+	// Whether the volume data is backed up successfully.
+	Result VolumeResult `json:"result,omitempty"`
+
 	CSISnapshotInfo          *CSISnapshotInfo          `json:"csiSnapshotInfo,omitempty"`
 	SnapshotDataMovementInfo *SnapshotDataMovementInfo `json:"snapshotDataMovementInfo,omitempty"`
 	NativeSnapshotInfo       *NativeSnapshotInfo       `json:"nativeSnapshotInfo,omitempty"`
 	PVBInfo                  *PodVolumeInfo            `json:"pvbInfo,omitempty"`
 	PVInfo                   *PVInfo                   `json:"pvInfo,omitempty"`
 }
 
+type VolumeResult string
+
+const (
+	VolumeResultSucceeded VolumeResult = "succeeded"
+	VolumeResultFailed    VolumeResult = "failed"
+	VolumeResultCancelled VolumeResult = "canceled"
```

Comment:
```suggestion
	VolumeResultCanceled VolumeResult = "canceled"
```
matching spelling of canceled on either side of `=`

Reply:
Modified.